iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/ConnectableSerDeUtil.java

```diff
@@ -164,6 +209,27 @@ public static void readOptionalPQ(Integer index, Terminal t, TreeDataReader read
                 .ifPresent(t::setQ);
     }
 
+    private static String indexToString(DcTerminal dcTerminal) {
+        if (dcTerminal.getSide() != null) {
+            return "" + dcTerminal.getSide().getNum();
+        } else if (dcTerminal.getTerminalNumber() != null) {
+            // is an AC/DC converter DC Terminal, the "_dc" portion is to be able to distinguish AC from DC part
+            return "_dc" + dcTerminal.getTerminalNumber().getNum();
```

Comment:
Since the other attributes are in camel case, using `"Dc"` instead of `"_dc"` would be more consistent.